configmap/hash-gen/main.go

```diff
@@ -49,7 +49,7 @@ func processFile(fileName string) error {
 		return err
 	}
 
-	if err := ioutil.WriteFile(fileName, out, 0644); err != nil {
+	if err := ioutil.WriteFile(fileName, out, 0600); err != nil {
```

Comment:
should we get this from an `os.Stat`?

Reply:
I don't see a need to copy permissions from somewhere else here.

Reply:
I think this is only relevant if the file doesn't already exist.

Reply:
Probably worth checking / adding a comment?

Reply:
Done. I rolled the change back, didn't seem important enough to fuzz about a lot tbh and this currently seems to work just fine.